interface-elements-for-web/articles/menu/responsive-layout/responsive-layout.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Responsive Layout
+author: Anna Kondratova
+---
+# Responsive Layout
+
+The menu can operate in the following modes when the browser window's width changes:
```

Comment:
The two sentences do not read like they are 'modes'.

Reply:
I still think it's the modes :) We can switch on/off each of these modes. Maybe the word "way" is better here?